src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: all 300ms ease;
```

```suggestion
    transition: color, transform 300ms ease;
```
Завжди краще специфікувати, до чого саме буде застосовуватсь transition